finality-aleph/src/party/mod.rs

```diff
@@ -432,9 +432,20 @@ where
     }
 
     pub async fn run(mut self) {
-        let last_finalized_number = self.client.info().finalized_number;
+        let mut finalized_number = self.client.info().finalized_number;
+        let mut previous_finalized_number;
+        loop {
+            sleep(Duration::from_secs(1)).await;
```

Comment:
Perhaps just name the magic number for better readability, e.g. EXPECTED_TIME_TO_NEW_FINALIZED_BLOCK ( could be longer ;) )

Reply:
Yeah, it sounds like an overkill. :) Since it's arbitrary and less than 1hr (or anything that can confuse a user) I leave it up to you.

Reply:
> (it's how long we sleep before checking if finalized_number has changed)

Oh, I only now noticed this, in this case it definitely should be below 1s – in normal operation we finalize 1 block/s, so this would slow down many nodes joining (they would eventually join because of random delays, but bleh). I would say something in the range 0.5-0.8s would be good.